### ISSUE TITLE
mkchromecast -t shows no icon on Plasma 5.14.5

### ISSUE THREAD

**Issue by CallumDev** [muammar/mkchromecast#242]
Using Arch Linux aur package, `python-pyqt5` is installed.

Console output:
```
Mkchromecast v0.3.8.1
Selected backend: /usr/bin/parec
Selected audio codec: mp3
Default bitrate used: 192k
Default sample rate used: 44100Hz.
```

**Comment by polyzen** [comment]
AUR package maintainer here. The tray icon works for me with just i3wm.

**Comment by CallumDev** [comment]
It has occurred to me I don't currently have a working Chromecast. Does it work on i3wm when there is no Chromecast present? If not then it's more of a documentation thing than a behaviour issue (though a no chromecast icon would be nice)

**Comment by polyzen** [comment]
The icon shows [this](https://ptpb.pw/eViW.png) when no casting device is found, even if you have neither of the casting deps (`python-pychromecast` or `python-soco`) installed.

FWIW I understand there is some issue with Qt5 tray icons. Another program I package with a Qt5 GUI just shows a blank space in the tray for me ^^.

**Comment by muammar** [comment]
@polyzen that is correct. I haven't had time to find a solution for that problem yet.

**Comment by djerius** [comment]
I had a similar symptom which turned out to be that the default icon is black and it was displaying against a black background, so was invisible.  I was able to find it by randomly right-clicking in the Gnome 3 panel and using the "Preferences" option to change the color to white. It can also be set using the `colors` attribute in  `~/.config/mkchromecast/mkchromecast.cfg`:

```
[settings]
colors = white
```

**Comment by oniGino** [comment]
I am having the same problem running on gentoo

**Comment by Anyeos** [comment]
LOL, same here, the icon is showing but I don't noticed it because I am using a Dark Theme on Gnome Shell. And the icon is black with the dark background and it is barely visible. But it is working and showing.
